### ISSUE TITLE
Error with version 1.2.22

### ISSUE THREAD

**Issue by AntonShevchuk** [postmanlabs/newman#281]
I have a problem with my Travis-CI build
I use newman for tests my REST API and with version 1.2.22 i received error

https://travis-ci.org/bluzphp/skeleton/jobs/84894208
```
/home/travis/.nvm/v0.10.36/lib/node_modules/newman/src/runners/RequestRunner.js:309
        reqData.forEach(function (paramData) {
                ^
TypeError: Object  has no method 'forEach'
    at Object.jsface.Class._parseFormParams (/home/travis/.nvm/v0.10.36/lib/node_modules/newman/src/runners/RequestRunner.js:309:17)
    at Object.jsface.Class._setBodyData (/home/travis/.nvm/v0.10.36/lib/node_modules/newman/src/runners/RequestRunner.js:254:44)
    at Object.jsface.Class._getRequestOptions (/home/travis/.nvm/v0.10.36/lib/node_modules/newman/src/runners/RequestRunner.js:240:8)
    at Object.jsface.Class._execute (/home/travis/.nvm/v0.10.36/lib/node_modules/newman/src/runners/RequestRunner.js:170:30)
    at null._onTimeout (/home/travis/.nvm/v0.10.36/lib/node_modules/newman/src/runners/RequestRunner.js:222:11)
    at Timer.listOnTimeout [as ontimeout] (timers.js:112:15)
```
similar build with old 1.2.21 is OK - https://travis-ci.org/bluzphp/skeleton/jobs/84894209

Collection is here - https://github.com/bluzphp/skeleton/blob/master/tests/postman/collection.json

I think problem with this rule:
```
{
			"id": "31d1ea31-6174-5a40-0d6c-23166914665c",
			"headers": "Accept: application/json\n",
			"url": "{{URL}}/pages/rest/1",
			"preRequestScript": "",
			"pathVariables": {},
			"method": "PUT",
			"data": [],
			"dataMode": "urlencoded",
			"version": 2,
			"tests": "tests[\"Status code is 405\"] = responseCode.code === 405;\ntests[\"Content-Type is present\"] = responseHeaders.hasOwnProperty(\"Content-Type\");\ntests[\"Content-Type is JSON\"] = responseHeaders[\"Content-Type\"] == \"application/json\";",
			"time": 1416235010649,
			"name": "update item",
			"description": "SHOULD return **403**   ",
			"collectionId": "09f9d7a9-98d0-d2f1-eeec-363c735aba39",
			"responses": [],
			"synced": false
		},
```

**Comment by ihti** [comment]
Im also having problems with 1.2.22 using newman as library, don't have time to explore in depth, just back to 1.2.21 and all good.
Looks like environment is ignored or data isn't send in POST.

**Comment by abhijitkane** [comment]
@ihti @AntonShevchuk We've released v1.2.23 which has a related fix. Can you check if it solves the issue?
I was able to reproduce this on 1.2.21 as well - not sure what caused the error @AntonShevchuk posted.

**Comment by AntonShevchuk** [comment]
Solved!
Thx!

Link to passed build - https://travis-ci.org/bluzphp/skeleton/jobs/86375124
